SeleniumAutomationUtils/SeleniumExtensions/WebDriverExtensions.cs

```diff
@@ -3509,6 +3510,11 @@ public static void PingDriver(this WebDriver driver)
         {
             driver.FindElement(By.XPath(".//body"));
         }
+        
+        public static void SetWindowSize(this WebDriver driver, int width, int height)

```

Comment:
if (width < 0 || height < 0)
    {
        throw new ArgumentException("Width and height must be non-negative values.");
    }